.github/workflows/backport.yaml

```diff
@@ -0,0 +1,45 @@
+name: Automated Backporting
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  backports:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    outputs:
+      releases: ${{ steps.checkBackport.outputs.releases }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: .github/scripts
+      - id: checkBackport
+        name: Check Backport
+        run: |
+          PR_NUMBER="${{ github.event.pull_request.number }}"
+          echo "PR:         https://github.com/$GITHUB_REPOSITORY/pull/$PR_NUMBER"
```

Comment:
Instead of recreating the url, you can use: `${{ github.event.pull_request.html_url }}`

Reply:
Thanks, I've amended the PR!